### PR TITLE
[desktop] Add release notes to AppStream .metainfo.xml

### PR DESCRIPTION
**Pull request by Ferenc-**
Signed-off-by: Ferenc Géczi <ferenc.gm@gmail.com>

### CODE REVIEW

**Comment by Ferenc-** [comment]
@biodranik with the current test trigger setup, any change to `qt/res/app.organicmaps.desktop.metainfo.xml ` also triggers,
the `osm_auth_tests` which will most probably fail as long as the service at https://master.apis.dev.openstreetmap.org/login?cookie_test=true is not useable, because `osm_auth_tests` rely on that.  
Possible options for working around:
1. Temporarily disable the trigger when only the  `qt/res/app.organicmaps.desktop.metainfo.xml ` is updated,
with a new `ignore` rule [here](https://github.com/organicmaps/organicmaps/blob/master/.github/workflows/linux-check.yaml#L5).
2. Temporarily disable the `osm_auth_tests` from [here](https://github.com/organicmaps/organicmaps/blob/master/editor/CMakeLists.txt#L48).
3. Trying to use an alternative to `master.apis.dev.openstreetmap.org`?

File: qt/res/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -107,6 +107,34 @@
...
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OSM maps data as of November 19</li>
+         </ul>
+       <p>Routing and Navigation:</p>
+         <ul>
+           <li>Corrected bicycle's speed for better routing</li>
+         </ul>
+       <p>Desktop:</p>
```
Comment (biodranik) [inline qt/res/app.organicmaps.desktop.metainfo.xml:120]: It doesn't make sense here and misleads users. Let's remove desktop word.

File: qt/res/app.organicmaps.desktop.metainfo.xml [hunk 2]
```diff
@@ -107,6 +107,34 @@
   </screenshots>
 
   <releases>
+   <release version="2022.11.25-2" date="2022-11-25">
```
Comment (biodranik) [inline qt/res/app.organicmaps.desktop.metainfo.xml:110]: We can tag the release today )

File: qt/res/app.organicmaps.desktop.metainfo.xml (hunk 1 above)
Comment (Ferenc-) [reply qt/res/app.organicmaps.desktop.metainfo.xml]: OK, I have removed the `Desktop` paragraph, and I have moved the content of the paragraph under `Highlights`.

File: qt/res/app.organicmaps.desktop.metainfo.xml (hunk 2 above)
Comment (Ferenc-) [reply qt/res/app.organicmaps.desktop.metainfo.xml]: OK, changed it to `2022-11-25`, the suffix is still `-2`, I have never followed how that is determined.

Comment (biodranik) [reply qt/res/app.organicmaps.desktop.metainfo.xml]: It's the number of commits in the day when the release/tag is created. It is generated/calculated by tools/unix/version.sh

**Comment by biodranik** [comment]
1. Adding appropriate ignore rules is a good idea. You can check if some other files should also be ignored.
2. It's better to contact those people who support dev OSM API. It would be a pity to release a non-working OSM login because we disabled tests.
3. What are the alternatives? Run dev osm server ourselves? It is theoretically possible if it's easy to set up and maintain. Are there other alternate dev servers that keep up-to-date with the master OSM branch?

**Comment by Ferenc-** [comment]
I have posted a question [here](
https://community.openstreetmap.org/t/why-is-master-apis-dev-openstreetmap-org-not-working/6001)

**Comment by Ferenc-** [comment]
https://github.com/openstreetmap/operations/issues/794

**Comment by biodranik** [comment]
Cool, looks like it was fixed. Thanks, @Ferenc- !

**Pull request closed without merging**